### PR TITLE
Update to support custom encodings

### LINKED ISSUE #58: Binary/Other encoding support
I noticed this only seems to support `utf-8` as the hardcoded encoding type when loading files with `fs`. Would it be possible to extend these functions to support a custom file encoding? 

I'm wanting to use this to read binary files as well but it seems the current encoding might be leading to broken results (file still reads but the bits are all out of order and can't be parsed using jDataView correctly to pull the Uint8 values).

Tested locally by modifying my local node_modules package of raw.macro and it appears to resolve my issue. Would love built in support for encoding choice!

### PR DESCRIPTION
**Pull request by cnnranderson**
Resolve #58

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "raw.macro",
   "description": "Read file contents at build time, similar to webpack raw-loader",
-  "version": "0.4.2",
+  "version": "0.5.0",
```
Comment (pveyes) [inline package.json]: I'll cut another version manually by myself

File: raw.macro.d.ts [hunk 2]
```diff
@@ -1,3 +1,3 @@
 declare module "raw.macro" {
-  export default function raw(rawPath: string): string;
+  export default function raw(rawPath: string, encoding?: string): string;
```
Comment (pveyes) [inline raw.macro.d.ts:2]: nice

File: src/__tests__/macro.test.js [hunk 3]
```diff
@@ -70,6 +70,13 @@ pluginTester({
       const a1 = raw(\`./\${fixtureDir}/markdown.md\`);
       const a2 = raw(\`./fixtures/\${fileName}\`);
     `,
+    "set encoding": `
```
Comment (pveyes) [inline src/__tests__/macro.test.js]: maybe "custom encoding" is more apt?

**Review by pveyes — CHANGES_REQUESTED** [review]
Please revert other files unrelated to the changes (package.json, package-lock, yarn.lock). Other than that it's great

Thanks for making this PR!

File: package.json (hunk 1 above)
Comment (cnnranderson) [reply package.json]: Fixed!

File: src/__tests__/macro.test.js (hunk 3 above)
Comment (cnnranderson) [reply src/__tests__/macro.test.js]: Agreed. Updated

**Review by pveyes — APPROVED** [review]
LGTM

**Pull request merged**